sklearn/tree/_criterion.pyx

```diff
@@ -199,6 +180,63 @@ cdef class Criterion:
                                  - (self.weighted_n_left /
                                     self.weighted_n_node_samples * impurity_left)))
 
+    cdef void pointer_reset(
+        self,
+        SIZE_t start,
+        SIZE_t end
+    ) nogil:
+        pass
```

Comment:
I would add a docstring here describing what the function is intended to do.

Reply:
Pushed new commits for this also 😃